src/Components/Events/Cards/index.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import {Text, StyleSheet, View, Linking} from 'react-native';
+import ScaledImage from '../../ScaledImage';
+import { withCard } from './../../../Decorators/Card';
+import Badge from './CardBadge';
+
+const EventCard = ({ props, isOver }) => {
+    return (
+     <View style={styles.card} >
+        <ScaledImage width={286}   source={props.highlights.source} />
+        <Text style={styles.title}>{props.title}</Text>
+        <Badge text={props.date} link={require('./../../../assets/events_and_highlights/calendar.png')}  />
+        <Badge text={props.location} link={require('./../../../assets/events_and_highlights/location.png')} />
+        <Badge text={props.timings} link={require('./../../../assets/events_and_highlights/time.png')} />
+        <View style={{marginTop: 32}}>
+            {props.description.map((detail,index) => (
```

Comment:
Use `index` as a key to the child when using `map` (refer the warnings in the console)

Reply:
good call with index as key.